buzsaki_lab_to_nwb/tingley_code_septal/tingleyseptalnwbconverter.py

```diff
@@ -0,0 +1,134 @@
+"""Authors: Heberto Mayorquin and Cody Baker."""
+import dateutil
+from pathlib import Path
+from copy import Error, deepcopy
+from collections import Counter
+from datetime import datetime
+
+from nwb_conversion_tools import (
+    NWBConverter,
+    NeuroscopeRecordingInterface,
+    NeuroscopeLFPInterface,
+    NeuroscopeSortingInterface,
+    CellExplorerSortingInterface,
+)
+
+from .tingleyseptalbehaviorinterface import TingleySeptalBehaviorInterface
+from .tingleyseptal_utils import read_matlab_file
+
+
+DEVICE_INFO = dict(
+    cambridge=dict(
+        name="Cambridge prob (1 x 64)",
+        description=(
+            "Silicon probe from Cambridge Neurotech. Electrophysiological data were "
+            "acquired using an Intan RHD2000 system (Intan Technologies LLC) digitized with20 kHz rate."
+        ),
+    ),
+    neuronexus_4_8=dict(
+        name="Neuronexus probe (4 x 8)",
+        description=(
+            "A 4 (shanks) x 8 (electrodes) silicon probe from Neuronexus. Electrophysiological data were "
+            "acquired using an Intan RHD2000 system (Intan Technologies LLC) digitized with 20 kHz rate."
+        ),
+    ),
+    neuronexus_5_12=dict(
+        name="Neuronexus probe (5 x 12)",
+        description=(
+            "A 5 (shanks) x 12 (electrodes) silicon probe from Neuronexus. Electrophysiological data were "
+            "acquired using an Intan RHD2000 system (Intan Technologies LLC) digitized with 20 kHz rate."
+        ),
+    ),
+    neuronexus_6_10=dict(
+        name="Neuronexus probe (6 x 10)",
+        description=(
+            "A 6 (shanks) x 10 (electrodes) silicon probe from Neuronexus. Electrophysiological data were "
+            "acquired using an Intan RHD2000 system (Intan Technologies LLC) digitized with 20 kHz rate."
+        ),
+    ),
+    neuronexus_8_8=dict(
+        name="Neuronexus probe (8 x 8)",
+        description=(
+            "A 8 (shanks) x 8 (electrodes) silicon probe from Neuronexus. Electrophysiological data were "
+            "acquired using an Intan RHD2000 system (Intan Technologies LLC) digitized with 20 kHz rate."
+        ),
+    ),
+    old_neuronexus_probe=dict(
+        name="Old neuronexus probe",
```

Comment:
```suggestion
    old_neuronexus_probe=dict(
        name="Neuronexus probe (4 x 1)",
```

For consistency with the other names; you can re-use it in the case where you see multiple